packages/react-components/src/types/ChatMessage.ts

```diff
@@ -21,129 +21,124 @@ export type MessageAttachedStatus = 'bottom' | 'top';
 export type MessageContentType = 'text' | 'html' | 'richtext/html' | 'unknown';
```

Comment:
This file has the key type rework. Everything else is a fairly straightforward porting over to the new types.